src/Orchard.Web/Core/Shapes/CoreShapes.cs

```diff
@@ -811,6 +814,54 @@ string PagerId
         public void DefinitionTemplate(HtmlHelper Html, TextWriter Output, string TemplateName, object Model, string Prefix) {
             RenderInternal(Html, Output, "DefinitionTemplates/" + TemplateName, Model, Prefix);
         }
+        [Shape]
+        public void Tab(dynamic Display, IGrouping<string, dynamic> Tab, TextWriter Output) {
```

Comment:
And now we can customize it, good (still necessity of an admin theme)

Reply:
We have a couple of those here https://github.com/LaserSrl/Laser.Orchard.Platform/tree/master/src/Themes . Not sure that they would cover all possible uses in the community, because they are developed for our own custom needs.